src/components/PropertyLookup.vue

```diff
@@ -53,9 +53,19 @@ export default Vue.extend( {
 			}
 		},
 		async searchPropertiesOnScroll( options: SearchOptions ): Promise<void> {
-			const searchResults = await this.$store.dispatch( 'searchProperties', options );
+			let searchResults = await this.$store.dispatch( 'searchProperties', options );
+			searchResults = this.setTagForSearchResults( searchResults );
+
 			this.searchResults = this.searchResults.concat( searchResults );
 		},
+		setTagForSearchResults( searchResults: SearchResult[] ): object[] {
```

Comment:
same observation as before: `SearchResult[]` instead of object[] is a better type in this case.